packages/passport/sdk/src/authManager.ts

```diff
@@ -37,15 +38,19 @@ const formUrlEncodedHeader = {
   },
 };
 
+const logoutEndpoint = '/oidc/logout';
+const authorizeEndpoint = '/?authorize';
+
 const getAuthConfiguration = (config: PassportConfiguration): UserManagerSettings => {
   const { authenticationDomain, oidcConfiguration } = config;
 
   const store = typeof window !== 'undefined' ? window.localStorage : new InMemoryWebStorage();
   const userStore = new WebStorageStateStore({ store });
 
-  let endSessionEndpoint = `${authenticationDomain}/v2/logout?client_id=${oidcConfiguration.clientId}`;
+  const endSessionEndpoint = new URL(logoutEndpoint, authenticationDomain);
+  endSessionEndpoint.searchParams.append('client_id', oidcConfiguration.clientId);
```

Comment:
This is a nitpick, but we can use `set` instead of `append`? If a particular query string parameter has already been set and `append` is used, we'll end up with two values, e.g:
```
const myUrl = new URL('/hello/world', 'https://example.com');
myUrl.searchParams.append('queryOne', 'someValue');
myUrl.searchParams.append('queryOne', 'someSecondValue');
myUrl.toString();
// 'https://example.com/hello/world?queryOne=someValue&queryOne=someSecondValue'
```